.github/workflows/daily-builds.yml

```diff
@@ -65,14 +65,14 @@ jobs:
           ninja src/all
           ccache --show-stats
           ./dragonfly --alsologtostderr --default_lua_flags=allow-undeclared-keys &
-      - name: Clone and build BullMQ
-        run: |
-          cd ${{github.workspace}}
-          git clone https://github.com/taskforcesh/bullmq.git
-          cd bullmq
-          yarn install --ignore-engines --frozen-lockfile --non-interactive
-          yarn build
-      - name: Test BullMQ with dragonfly
-        run: |
-          cd ${{github.workspace}}/bullmq
-          yarn test -i -g "should process delayed jobs with several workers respecting delay"
+      # - name: Clone and build BullMQ
+      #   run: |
+      #     cd ${{github.workspace}}
+      #     git clone https://github.com/taskforcesh/bullmq.git
+      #     cd bullmq
+      #     yarn install --ignore-engines --frozen-lockfile --non-interactive
+      #     yarn build
+      # - name: Test BullMQ with dragonfly
+      #   run: |
+      #     cd ${{github.workspace}}/bullmq
+      #     yarn test -i -g "should process delayed jobs with several workers respecting delay"
```

Comment:
commenting out this line is enough

Reply:
yeah, but then the previous step (building bullmq) would be of no use. That's why I commented it too.